### DISCUSSION TITLE
Getting the message "CorsairLink could not initialize or has no sensors", does anyone know how to fix?

### DISCUSSION THREAD

**Discussion by georgewashingmachine** [EvanMulawski/FanControl.CorsairLink#12] [Q&A] [answered]
Hi there.

I'm trying to get control of my CPU fans as they are currently connected to the CPU commander but I'm receiving the message "CorsairLink could not initialize or has no sensors" when refreshing my fan sensors after rebooting FanControl and refreshing my sensors.

I have a Corsair iCUE H150i Elite RGB with 3 fans.

Thanks.

**Comment by EvanMulawski** [comment, marked as answer]
Hi @georgewashingmachine, support for the Hydro series coolers is in progress but not supported in v1.0.0. Please see #9. If you would like to help test this, please download the latest build in the comments for that issue; provide your feedback in that issue with the Fan Control `log.txt` file attached. Thanks!

**Reply by georgewashingmachine** [reply]
Okaydokes no worries, thanks!

**Reply by EvanMulawski** [reply]
A beta release is now available that adds support for this device.

https://github.com/EvanMulawski/FanControl.CorsairLink/releases/tag/v1.1.0-beta.1
